docs/source/usersguide/scripts.rst

```diff
@@ -0,0 +1,279 @@
+.. _usersguide_scripts:
+
+=======================
+Executables and Scripts
+=======================
+
+.. _scripts_openmc:
+
+----------
+``openmc``
+----------
+
+Once you have a model built (see :ref:`usersguide_basics`), you can either run
+the openmc executable directly from the directory containing your XML input
+files, or you can specify as a command-line argument the directory containing
+the XML input files. For example, if your XML input files are in the directory
+``/home/username/somemodel/``, one way to run the simulation would be:
+
+.. code-block:: sh
+
+    cd /home/username/somemodel
+    openmc
+
+Alternatively, you could run from any directory:
+
+.. code-block:: sh
+
+    openmc /home/username/somemodel
+
+Note that in the latter case, any output files will be placed in the present
+working directory which may be different from ``/home/username/somemodel``. If
+you're using the Python API, :func:`openmc.run` is equivalent to running
```

Comment:
This sentence about `openmc.run` seems a little out of place since it is enveloped by info about the `openmc` executable. Perhaps put this sentence in a tip block at the end of this subsection?